src/nodejs/neon-js/cli/rpc/getBestBlockHash.js

```diff
@@ -8,31 +8,57 @@ require('module-alias/register')
 const program = require('commander')
 const _       = require('underscore')
 
-const neon      = require('@cityofzion/neon-js')
+const neon    = require('@cityofzion/neon-js')
 const dbg     = require('nodejs_util/debug')
+const netutil = require('nodejs_util/network')
+
+var cfg       = require('nodejs_config/config.js')
+var config    = cfg.load('nodejs_config/nodejs.config.json')
+
+let nodes = []
+let defly = false
 
 function print(msg) {
   console.log(msg);
 }
 
 program
-  .version('0.1.0')
-  .usage('-n <node>')
+  .version('0.2.0')
+  .usage('')
   .option('-d, --debug', 'Debug')
-  .option('-n, --node <node>', 'set RPC node to use (be sure to preface with https://)')
+  .option('-n, --node [node]', 'set RPC node to use (be sure to preface with https://), if not provided will try to use node with tallest block')
+  .option('-N, --Net [Net]', 'Select network [net]: i.e., TestNet or MainNet', 'TestNet')
 
   .parse(process.argv);
 
-if (!program.node) {
-  program.help()
+if (program.debug) {
+  print('DEBUGGING: ' + __filename)
+  defly = true
 }
 
-if (program.debug) {
-  print('DEBUGGING');
+if (!program.node) {
+  // get a node from the list and try it
+  let net = netutil.resolveNetworkId(program.Net)
+
+  nodes = cfg.get_nodes(net)
+
+  if (defly) dbg.logDeep('config nodes: ', nodes)
+
+  netutil.getNodesByTallest(nodes).then(rankedNodes => {
+    if (defly) dbg.logDeep('sorted nodes: ', rankedNodes)
+    nodes = rankedNodes
```

Comment:
do you always want to override nodes instead of pushing it?

Reply:
If none is specified, yes.